spec/pages/manual_page_spec.rb

```diff
@@ -33,8 +33,8 @@
       source/manual/howto-merge-a-pull-request-from-an-external-contributor.html.md
     ])
       it "doesn't use H1 tags (the page title is already an H1)" do
-        expect(raw).not_to match(/\n#\s/), "This page contains an unnecessary H1." \
-          " This may have been triggered because of a # comment in a code block"
+        raw_without_code_blocks = raw.remove(/```[a-z]*\n[\s\S]*?\n```/) # want to allow `#` (comments) in code blocks
```

Comment:
FYI regex taken from https://coderwall.com/p/r6b4xg/regex-to-match-github-s-markdown-code-blocks